python/utils/bootstrap_utilities.py

```diff
@@ -107,6 +108,73 @@ def _ensure_no_unicode(obj):
         return obj
 
 
+def _enumerate_per_line(items):
+    """
+    Enumerate all items from an array, one line at a time.
+
+    For example,
+        - one
+        - two
+        - three
+
+    :returns: The formatted output.
+    """
+    return "\n".join("- {}".format(item) for item in items)
+
+
+def _env_not_set_or_split(var_name):
+    """
+    Format a PATH-like environment variable for output.
+
+    :param str var_name: Name of the env var.
+    :returns: "Not Set" if variable is not set, a bullet list otherwise.
+    """
+    if var_name not in os.environ:
+        return "Not Set"
+    else:
+        # Add a \n before the first item so each item in the output start from the
+        # beginning of the time. Otherwise you'd get.
```

Comment:
I'm having déjà vu 😛 

```suggestion
        # beginning of the line. Otherwise you'd get.
```